OCR_grocery.py

```diff
@@ -27,20 +27,30 @@ def spellchecker(misspelled_word='caaar',n_gram = 1):
     return min_word
 
 
+def ingr_splitter(item):
+    search_obj = re.search('\d+', item)
+    if (search_obj != None):
+        quantity = item[search_obj.start() : search_obj.end()]
+        ingredient = item[search_obj.end():]
+    else:
+        quant = 1
```

Comment:
at `if` statement we have `quantity` but at `else` we have `quant` is this is a mistake? please advice